### PR TITLE
2021 q1 changes

### PR DESCRIPTION
**Pull request by lewisrenfrew**
- this might not mean much to anyone else. I could show and tell it if anyone is interested.

### CODE REVIEW

File: src/Domain/VatReturnCalculationService.cs [hunk 1]
```diff
@@ -184,9 +185,25 @@ from nominal_ledger
...
-            var decimalResult = res == DBNull.Value ? 0 : decimal.Parse(res.ToString());
-            return decimalResult + this.GetCanteenTotals()["vat"];
+            var result = this.databaseService.ExecuteQuery(sql);
+            
+            return (from DataRow row in result.Tables[0].Rows
+                    select row.ItemArray
+                    into values
+                    select new NominalLedgerEntry 
+                               {
+                                   Tref = Convert.ToInt32(values[0]),
```
Comment (richardiphillips) [inline src/Domain/VatReturnCalculationService.cs:195]: personally I prefer int.parse and decimal.parse.

File: src/Facade/Services/VatReturnService.cs [hunk 2]
```diff
@@ -54,13 +54,22 @@ public IResult<CalculationValuesResource> GetCalculationValues()
...
+                            }),
                         PurchasesGoodsTotal = this.calculationService.GetPurchasesTotals()["goods"],
                         PurchasesVatTotal = this.calculationService.GetPurchasesTotals()["vat"],
-                        IntrastatArrivalsGoodsTotal = this.calculationService.GetIntrastatArrivals()["goods"],
-                        IntrastatArrivalsVatTotal = this.calculationService.GetIntrastatArrivals()["vat"],
-                        InstrastatDispatchesGoodsTotal = 0 // didn't manage to obtain this value, still needs to be entered for now
-                });
+                        IntrastatArrivalsGoodsTotal = this.calculationService.GetIntrastatArrivals()["goods"], // should be 0 post brexit
+                        IntrastatArrivalsVatTotal = this.calculationService.GetIntrastatArrivals()["vat"], // should be 0 post brexit
+                        InstrastatDispatchesGoodsTotal = 0,
```
Comment (richardiphillips) [inline src/Facade/Services/VatReturnService.cs:71]: ,

File: src/Service.Host/package.json [hunk 3]
```diff
@@ -30,75 +30,77 @@
...
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-hot-loader": "^4.12.18",
-    "react-redux": "^7.1.3",
-    "react-router": "^5.1.2",
-    "react-router-dom": "^5.1.2",
-    "redux": "^4.0.4",
-    "redux-api-middleware": "^3.0.1",
-    "redux-devtools": "^3.5.0",
+    "react": "^17.0.2",
```
Comment (richardiphillips) [inline src/Service.Host/package.json:92]: that's great you've tested all these upgrades. I'd started looking at them for stores but you saved me a job.

**Review by richardiphillips — APPROVED** [review]
show and tell would be good.

File: src/Service.Host/package.json (hunk 3 above)
Comment (lewisrenfrew) [reply src/Service.Host/package.json:92]: I'm a bit suspicious about how painless it was... Might be a bit more to it in more complicated projects

**Pull request merged**